umbraco-cloud/begin-your-cloud-journey/the-cloud-portal/organizations/organization-login-providers.md

```diff
@@ -0,0 +1,316 @@
+
+For example, if the roles claim is called `user_roles` in your provider, you set the **User Mapping Claim Name** to `user_roles`.
+
+## Signing in using the Login Provider
+
+When trying to access Umbraco Cloud Portal through `s1.umbraco.io`, you are greeted by an Umbraco ID sign-in screen. 
+
+To sign in with your login provider, you must use a special sign-in URL that is unique to your Login Provider.
+
+1. Go back to Cloud Portal, where you registered the Login Provider.
+2. Click on the `Sign-in and Redirect URLs` button.
+
+<figure><img src="../../../.gitbook/assets/organization-elp-signin-url.gif" alt=""><figcaption>How to retrive the Sign in Url</figcaption></figure>
+
+3. Give the URL to the Organization members you want to sign in using your Login Provider.
+
+## Project Permissions
+
+Project Permissions lets you set up access to Projects in the Portal while signed in with your Login Provider.
+
+You must add one Project Permission model per Project and one per Login Provider. It is not required to add Project Permissions to all projects. Projects without a Project Permissions tied to a Login Provider will not be shown to a user logged in with that particular Login Provider.  
+
+<figure><img src="../../../.gitbook/assets/organization-elp-project-permission-screen.png" alt=""><figcaption>Project Permission Screen</figcaption></figure>
+
+To set up Project Permission, follow these steps:
+
+1. Select a Project on the left side of the screen.
+2. Click on "+ Add" on the Login Provider you want to add Project Permissions for.
+
+<figure><img src="../../../.gitbook/assets/organization-elp-project-permission-add.png" alt=""><figcaption>Add Project Permission</figcaption></figure>
+
+3. Fill in the fields in the modal:
+  - Default Access Level (required)
+  - No Claim Found Behavior (required)
+  - User Mapping Claim Name
+  - Project User Mappings
+    - Consists of two fields: "Provider Role Value" and "Project Access Level"
+
+## How to fill in the Project Permissions
+
+### Default Access Level
+
+Select the level of access you want users to get for this project.
+
+The dropdown has two possible permissions:
+
+- Read Only
+- Read and Write
+
+#### Read
+
+A team member with Read permissions can only view the project in the portal and the backoffice. They are not able to deploy or change anything on the project itself.
+
+#### Read And Write
+
+A team member with Write permissions can do everything on a project except delete it and edit the team. A user with Write permissions can deploy changes between environments through the portal.
+
+This value is meant to be a fallback value and can be overwritten by "Project User Mappings" setting.
+If there are no Mappings available for the user the "No Claim Found Behavior" setting will evaluate if this fallback Permissions is used or "NoAccess".
```

Comment:
Instead of having this information here, wouldn't it be better to link over to the article we have about these roles already? 🤔  This is the article: https://docs.umbraco.com/umbraco-cloud/begin-your-cloud-journey/project-features/team-members#team-member-user-permissions

Reply:
Hi @sofietoft its a good point, but we choose to add our own section, mainly as we currently differe a bit between the default roles does and what will happen if you use the External Login Provider - It is because we are in an early stage, but i.e. backoffice, git and kudu access wont be granted. The article we already hae on team-member-user-permissions says that those things will be granted.

So to have clear boundries on what can be done when signing in with an External Login Provider, we "had" to add our own.

Reply:
Ah, that makes sense! 🙏